Model/Quote/SaveOrderBeforeSalesModelQuoteObserver.php

```diff
@@ -93,7 +93,7 @@ public function execute(Observer $observer)
 
         if ($quote->hasData(Checkout::FIELD_DELIVERY_OPTIONS) && $this->hasMyParcelDeliveryOptions($quote)) {
             $jsonDeliveryOptions = $quote->getData(Checkout::FIELD_DELIVERY_OPTIONS);
-            $deliveryOptions     = json_decode($jsonDeliveryOptions, true) ?? [];
+            $deliveryOptions     = $jsonDeliveryOptions ? json_decode($jsonDeliveryOptions, true, 512, JSON_THROW_ON_ERROR) : [];
```

Comment:
JSON_THROW_ON_ERROR is pas vanaf php 7.3 en wij ondersteunen dacht ik 7.2 nog?

Reply:
Kan je niet bij $quote->getData zorgen dat $jsonDeliveryOptions niet null kan zijn? Dat maakt deze regel minder lang / lelijk / moeilijk.